pandas/core/indexes/interval.py

```diff
@@ -192,6 +214,8 @@ class IntervalIndex(IntervalMixin, Index):
     # Immutable, so we are able to cache computations like isna in '_mask'
     _mask = None
 
+    _raw_inherit = {"_ndarray_values", "__array__", "overlaps", "contains"}
```

Comment:
same as above, should we also add this attribute as an empty dict in Index?

Reply:
id rather not have a dummy attr on Index, but im building up to implementing ExtensionIndex which will have it

Reply:
see my comment below